sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -415,7 +417,7 @@ class RecognizePiiEntitiesResult(DictMixin):
     :vartype entities:
         list[~azure.ai.textanalytics.PiiEntity]
     :ivar str redacted_text: Returns the text of the input document with all of the PII information
-        redacted out. Only returned for API versions v3.1 and up.
+        redacted out.
```

Comment:
Should we add in the object's main docstring to say that it's only returned for API versions v3.1 and up?

Reply:
I thought about this and think that we should make it clear that there is only one spot to check if the type/model/method is added in a new version -- the `versionadded` directive. I worry if we have it in too many spots, we might miss adding it in some places and then it will be confusing for users. :( what do you think?

Reply:
agreed